### DISCUSSION TITLE
Profile navbar height

### DISCUSSION THREAD

**Discussion by Ik1497** [requarks/wiki#6165] [Error / Bug Report]
The current navbar on the profile pages seems to be the wrong height, the height is currently auto determined, but I think it should be 100%

**How it currently is:**

![image](https://user-images.githubusercontent.com/113038466/219099474-83b1b855-fa65-4ba3-8cb4-364eba5d939b.png)

**How I think it should be:**

![image](https://user-images.githubusercontent.com/113038466/219099457-982a869a-2c3e-473e-8986-65bbd5dd31f6.png)

**Comment by NGPixel** [comment]
Which browser are you using? I can't replicate your issue.

**Reply by Ik1497** [reply]
I'm using Google Chrome. The wiki is located at https://wiki.streamer.bot if you'd like to test
